src/info-label.scss

```diff
@@ -0,0 +1,128 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+/*!
+.fd-badge+(( --success | --warning | --error ), (--pill | --filled))
+*/
+$block: #{$fd-namespace}-info-label;
+
+.#{$block} {
+  /*!
+  the below hardcoded colour values will be changed once we recieve variable creation from the fundamental team
+  */
+  $sapAccend1: #fdf3e6;
+  $sapAccend2: #f9e6e6;
```

Comment:
Maybe lets use here nice map structure and each loop, like:

```
  $color-accents: (
    "1": ("background": #fdf3e6, "border": var(--sapAccentColor1)),
    "2": ("background": #f9e6e6, "border": var(--sapAccentColor2)),
    "3": ("background": #fce9f2, "border": var(--sapAccentColor3)),
    ...
  );

  @each $set-name, $color-set in $color-accents {
    &--accent-color-#{$set-name} {
      background-color: map_get($color-set, "background");
      border-color: map_get($color-set, "border");
    }
  }
```

Reply:
fixed